velox/docs/configs.rst

```diff
@@ -116,11 +116,11 @@ Expression Evaluation Configuration
      - true
      - If set, cast from string to date allows only ISO 8601 formatted strings: [+-](YYYY-MM-DD).
        Otherwise, allows all patterns supported by Spark:
-         * ``[+-]yyyy*``
-         * ``[+-]yyyy*-[m]m``
-         * ``[+-]yyyy*-[m]m-[d]d``
-         * ``[+-]yyyy*-[m]m-[d]d *``
-         * ``[+-]yyyy*-[m]m-[d]dT*``
+       ``[+-]yyyy*``,
+       ``[+-]yyyy*-[m]m``,
+       ``[+-]yyyy*-[m]m-[d]d``,
+       ``[+-]yyyy*-[m]m-[d]d *``,
+       ``[+-]yyyy*-[m]m-[d]dT*``
```

Comment:
Showing as a bullet list is probably more readable then comma-separated list. 

Also, should we include (BC) patterns?

Reply:
@mbasmanova  (BC) pattern is a special case of `[+-]yyyy*-[m]m-[d]d *`

Reply:
Interesting. I guess I do not understand what "*" in the pattern means. Would you clarify? Let's also update the docs to make sure other readers are not misled.

Reply:
The documents are copied from Spark's comment https://github.com/apache/spark/blob/3e5203c64c06cc8a8560dfa0fb6f52e74589b583/sql/api/src/main/scala/org/apache/spark/sql/catalyst/util/SparkDateTimeUtils.scala#L286-L298

I think the meaning of `*` in Spark's comment is any numbers, like `yyyy*`. But Spark doesn't check whether the trailing characters after 'T' or space are numbers, so in fact any characters are allowed.

Reply:
> Spark doesn't check whether the trailing characters after 'T' or space are numbers, so in fact any characters are allowed.

I see... Let's clarify this in the doc and provide examples. This is non-obvious.